src/SourceGenerators/Uno.UI.Tasks/ResourcesGenerator/UnoPRIResourcesWriter.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Uno.UI.Tasks.ResourcesGenerator
+{
+	internal class UnoPRIResourcesWriter
+	{
+		internal static void Write(string language, Dictionary<string, string> resources, string actualTargetPath, string comment)
+		{
+			using (var file = File.OpenWrite(actualTargetPath))
+			{
+				using (var writer = new BinaryWriter(file, Encoding.UTF8))
+				{
+					writer.Write(new byte[] { 0x75, 0x6E, 0x6F }); // Magic
```

Comment:
:D

Reply:
I don't understand :(

Reply:
#jeromelaban's magic ⭐️

Reply:
Explanation: <http://www.asciitable.com/>
- `0x75`: `u`
- `0x6E`: `n`
- `0x6F`: `o`

That's just a header to confirm we're dealing with the right file type. Like `PK` (`0x4B50`) for a _zip_ file (PK for _Phil Katz_, founder of PKWare) or `MZ` (0x4D5A) for a MS-DOS executable file (MZ for _Mark Zbikowski_, a Microsoft achitect).  Following this logic, it should have been `0x4A4C` (for `JL`).